inst/tests/tests.Rraw

```diff
@@ -17435,3 +17435,22 @@ exid = data.table(id=1, expected)
 test(2182.3, melt(DTid, measure.vars=list(a=c(NA,1), b=2:3), id.vars="id"), exid)
 test(2182.4, melt(DTid, measure.vars=list(a=c(NA,"a2"), b=c("b1","b2")), id.vars="id"), exid)
 test(2182.5, melt(DT.wide, measure.vars=list(a=c(NA,1), b=2:3), na.rm=TRUE)[, .(a, b)], data.table(a=2, b=2))#not testing variable because it is not computed correctly, #4455
+
+# gmin/gmax + NA values + na.rm, #4444
+if (test_bit64) {
+  DT = data.table(grp=c(1L, 1L, 1L, 2L), i64=as.integer64(c(NA, 1:3)))
+  old = options(datatable.optimize=2L)
+  test(2183.1, DT[ , min(i64), by=grp], data.table(grp=1:2, V1=as.integer64(c(NA, 3))))
+  test(2183.2, DT[ , min(i64, na.rm=TRUE), by=grp], data.table(grp=1:2, V1=as.integer64(c(1, 3))))
+  test(2183.3, DT[ , max(i64), by=grp], data.table(grp=1:2, V1=as.integer64(c(NA, 3))))
+  test(2183.4, DT[ , max(i64, na.rm=TRUE), by=grp], data.table(grp=1:2, V1=as.integer64(2:3)))
+  # force an all-NA group
+  DT[ , i64 := rev(i64)]
+  i64_default_min = suppressWarnings(min(as.integer64(NA), na.rm=TRUE))
+  i64_default_max = suppressWarnings(max(as.integer64(NA), na.rm=TRUE))
```

Comment:
I would add tests for that as well so in case bit64 will change this behavior we will be notified

Reply:
good idea. actually it turns out they _aren't_ warning, which differs from the equivalent base behavior. So the test is a bit wonky, WDYT?